route53/README.md

```diff
@@ -0,0 +1,42 @@
+Route53 for `libdns`
+=======================
+
+[![godoc reference](https://img.shields.io/badge/godoc-reference-blue.svg)](https://pkg.go.dev/github.com/libdns/route53)
+
+This package implements the [libdns interfaces](https://github.com/libdns/libdns) for AWS [Route53](https://aws.amazon.com/route53/).
+
+## Authenticating
+
+This package supports all the credential configuration methods described in the [AWS Developer Guide](https://docs.aws.amazon.com/sdk-for-go/v1/developer-guide/configuring-sdk.html), such as `Environment Variables`, `EC2 Instance Profile` and the `AWS Credentials file` located in `.aws/credentials`
+
+The following IAM policy is a minimal working example to give Caddy permissions to manage DNS records:
```

Comment:
One cool thing is that these libdns packages are totally unrelated to Caddy, so we don't need to make any references to Caddy; they can also be used by other code bases.

Reply:
Good point! Going to remove that reference